common/hyperscaler/rules/signature_set.go

```diff
@@ -0,0 +1,30 @@
+package rules
+
+type SignatureSet struct {
```

Comment:
Is this nesting necessary?
`SignatureSet.items`?
Why this is called `set`?

Reply:
It is a set of signatures. I have changed Signature* methods name in https://github.com/ralikio/kyma-environment-broker/commit/3351ed92a57a5e0657642f11564083d3fd449540. Let's talk to clarify your questions.

Reply:
This is map of slices. How come it is set?
Is it anyhow related to set ADT?

Reply:
No comments. Please propose a change.